Add TC Remove Azure Module and Generate Code

an-phung opened the pull request.

an-phung committed:
Add TC Remove Azure Module and Generate Code

giangchau95 commented on swtbot_test/src/testsuites/ChangeRTOSVersion.java:
```diff
@@ -37,18 +40,12 @@ public static void beforeClass() throws Exception {
 	}
 	
 	@Test
 	public void tc_01_CreateUsbxProject() throws Exception{
```
E: the TC only checks for components after changing the version, it does not show that rtos version is actually changed.
Need to check version too.

an-phung replied:
Fixed in latest commit

giangchau95 commented on swtbot_test/src/utilities/Utility.java:
```diff
@@ -142,11 +142,31 @@ public static void addComponentAndGenerate(String componentName) {
 		bot.sleep(15000);
 	}
 	
+	public static void removeComponentAndGenerate(String componentName) {
```
E: separate this utility into two different utils for removing components and generating code.

an-phung replied:
Fixed in latest commit

giangchau95 commented on swtbot_test/src/testsuites/RemoveAzureModuleAndGenerateCode.java:
```diff
@@ -0,0 +1,228 @@
+		Utility.addComponentAndGenerate("netx duo addons");
+		Utility.getProjectExplorerView().setFocus();
+		SWTBotTreeItem project = bot.tree().getTreeItem(projectModelSpecific.getProjectName() + " ["+ projectModelSpecific.getActiveBuildConfiguration() +"]");
+		project.getNode(ProjectParameters.FolderAndFile.FOLDER_LIBS).expand();
```
E: too complicated, no need to migrate 100% from rcptt, check for component added in component tree is enough

an-phung replied:
Fixed in latest commit

giangchau95 reviewed (commented):
Review-Time: 10m

an-phung committed:
Check for component in add and remove instead of File
Separate add remove component utils with click generate code
check for RTOS version in changeRTOSVersion

giangchau95 reviewed (approved):
Review-Time: 3m
